CoCafe/Views/CartTotalView.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  CartTotalView.swift
+//  CoCafe
+//
+//  Created by t2023-m0072 on 11/25/24.
+//
+import UIKit
+final class CartTotalView: UIView {
+    // 브라운 배경을 넣어주는 뷰 추가
+    private let infoContainerView: UIStackView = {
+        let view = UIStackView()
```

Comment:
변수 이름에서 스택뷰인것을 알 수 있으면 좋겠습니다!
저는 보통 스택뷰는 stview로 두는 편입니다☺️

Reply:
저도 비슷한 생각인데요, 아래와 같은 형식도 괜찮을 것 같네요!
```suggestion
        let stackView = UIStackView()
```